packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -1000,10 +1024,12 @@ class DataGrid extends React.Component {
       ariaStyles['aria-hidden'] = true;
     } else if (isPinned) {
       ariaStyles.id = `${id}-Pinned-Row-${row.id}-Section-${section.id}`;
-      ariaStyles['aria-owns'] = `${id}-Overflow-Row-${row.id}-Section-${section.id}`;
+      // ariaStyles['aria-owns'] = `${id}-Overflow-Row-${row.id}-Section-${section.id}`;
```

Comment:
Is this still needed?

Reply:
If we don't comment this line out, screen readers cannout read the overflow columns. So it seems that this is actively messing with the intention of these codings. I meant to leave a comment above this one explaining why we commented it out.